### PR TITLE
Update Avatar's Spec.md and Migration.md

### PR DESCRIPTION
**Pull request by behowell**
## Current Behavior

Avatar has changed since the initial spec and migration guide were written.

## New Behavior

Update Avatar's Spec.md and Migration.md to match the current state of Avatar.

## Related Issue(s)

* Part of #21237

### CODE REVIEW

File: packages/react-avatar/SPEC.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 The Avatar component represents a person or entity. It displays the person's image, initials, or an icon, and can be either circular or square.
 
-Note: The Avatar control has been mostly implemented already. Visit [Avatar Storybook Examples](http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/heads/master/react-avatar/storybook/index.html) to see the current state of the implementation.
+Note: The Avatar control has been mostly implemented already. Visit [Avatar Storybook Examples](https://fluentuipr.z22.web.core.windows.net/pull/21679/react-components/storybook/index.html?path=/docs/components-avatar--default) to see the current state of the implementation.
```
Comment (khmakoto) [inline packages/react-avatar/SPEC.md:9]: Should we use `heads/master` instead of a specific PR?
```suggestion
Note: The Avatar control has been mostly implemented already. Visit [Avatar Storybook Examples](https://fluentuipr.z22.web.core.windows.net/heads/master/react-components/storybook/index.html?path=/docs/components-avatar--default) to see the current state of the implementation.
```

File: packages/react-avatar/SPEC.md [hunk 2]
```diff
@@ -35,25 +35,34 @@ Both components support showing a badge to indicate status, but they have differ
...
+<Avatar size={72} name="Mona Kane" image={{ src: './MonaKane.jpg' }} />
+```
+
+Display an icon only:
+
+```jsx
+<Avatar aria-label="Team" icon={<PeopleTeamRegular />} shape="square" />
+```
+
+Displaying a badge:
```
Comment (khmakoto) [inline packages/react-avatar/SPEC.md:56]: nit: To be consistent above this should read `Display` instead of `Displaying`
```suggestion
Display a badge:
```

File: packages/react-avatar/SPEC.md [hunk 3]
```diff
@@ -271,11 +285,22 @@ See [MIGRATION.md](https://github.com/microsoft/fluentui/blob/master/packages/re
...
 ## Accessibility
 
-_TODO_
+The Avatar presents as a single img element to accessibility tools, regardless of what it is displaying (image, initials, or icon).
+
+- The Avatar's root has `role="img"`
+- All other slots have `aria-hidden="true"`.
+- The `<img>` additionally has `alt="" role="presentation"`
+
+The Avatar root's label will be determined using the following priority:
```
Comment (khmakoto) [inline packages/react-avatar/SPEC.md:299]: nit: Use present tense
```suggestion
The Avatar root's label is determined using the following priority:
```

**Review by khmakoto — APPROVED** [review]

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                                                    |    `44.89 kB`<br />`13.02 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `165.744 kB`<br />`46.848 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `32.479 kB`<br />`10.625 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/34503c0c1184cfbb966d58381381da51107ff7a9'>34503c0c1184cfbb966d58381381da51107ff7a9</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/ContextualMenu-mount_base.html">17010</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/ContextualMenu-mount.html">8644</a></td>
            <td>1000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/ContextualMenu-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Avatar-mount_base.html">888</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Avatar-mount.html">902</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/BaseButton-mount_base.html">968</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/BaseButton-mount.html">969</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Breadcrumb-mount_base.html">2774</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Breadcrumb-mount.html">2704</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ButtonNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/ButtonNext-mount_base.html">511</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/ButtonNext-mount.html">494</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Checkbox-mount_base.html">1622</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Checkbox-mount.html">1602</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/CheckboxBase-mount_base.html">1392</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/CheckboxBase-mount.html">1377</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/ChoiceGroup-mount_base.html">4933</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/ChoiceGroup-mount.html">4966</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/ComboBox-mount_base.html">1049</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/ComboBox-mount.html">1022</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/CommandBar-mount_base.html">10597</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/CommandBar-mount.html">10639</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/ContextualMenu-mount_base.html">17010</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/ContextualMenu-mount.html">8644</a></td>
            <td>1000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/ContextualMenu-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/DefaultButton-mount_base.html">1222</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/DefaultButton-mount.html">1167</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/DetailsRow-mount_base.html">3922</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/DetailsRow-mount.html">3876</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/DetailsRowFast-mount_base.html">3865</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/DetailsRowFast-mount.html">3936</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/DetailsRowNoStyles-mount_base.html">3776</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/DetailsRowNoStyles-mount.html">3672</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Dialog-mount_base.html">2332</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Dialog-mount.html">2359</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/DocumentCardTitle-mount_base.html">199</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/DocumentCardTitle-mount.html">193</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Dropdown-mount_base.html">3373</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Dropdown-mount.html">3433</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/FluentProviderNext-mount_base.html">1980</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/FluentProviderNext-mount.html">1986</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/FluentProviderWithTheme-mount_base.html">171</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/FluentProviderWithTheme-mount.html">174</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/FluentProviderWithTheme-virtual-rerender_base.html">115</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/FluentProviderWithTheme-virtual-rerender.html">121</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">209</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount.html">204</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/FocusTrapZone-mount_base.html">1939</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/FocusTrapZone-mount.html">1960</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/FocusZone-mount_base.html">1923</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/FocusZone-mount.html">1903</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/IconButton-mount_base.html">1856</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/IconButton-mount.html">1861</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Label-mount_base.html">394</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Label-mount.html">393</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Layer-mount_base.html">3074</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Layer-mount.html">3099</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Link-mount_base.html">509</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Link-mount.html">511</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/MakeStyles-mount_base.html">1771</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/MakeStyles-mount.html">1766</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/MenuButton-mount_base.html">1570</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/MenuButton-mount.html">1573</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/MessageBar-mount_base.html">2075</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/MessageBar-mount.html">2083</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Nav-mount_base.html">3415</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Nav-mount.html">3523</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/OverflowSet-mount_base.html">1178</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/OverflowSet-mount.html">1176</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Panel-mount_base.html">2243</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Panel-mount.html">2189</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Persona-mount_base.html">912</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Persona-mount.html">856</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Pivot-mount_base.html">1565</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Pivot-mount.html">1593</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/PrimaryButton-mount_base.html">1312</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/PrimaryButton-mount.html">1377</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Rating-mount_base.html">7978</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Rating-mount.html">8130</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/SearchBox-mount_base.html">1481</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/SearchBox-mount.html">1406</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Shimmer-mount_base.html">2625</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Shimmer-mount.html">2707</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Slider-mount_base.html">2025</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Slider-mount.html">2049</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/SpinButton-mount_base.html">5130</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/SpinButton-mount.html">5162</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Spinner-mount_base.html">484</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Spinner-mount.html">467</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/SplitButton-mount_base.html">3307</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/SplitButton-mount.html">3247</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Stack-mount_base.html">568</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Stack-mount.html">562</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/StackWithIntrinsicChildren-mount_base.html">2392</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/StackWithIntrinsicChildren-mount.html">2379</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/StackWithTextChildren-mount_base.html">5405</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/StackWithTextChildren-mount.html">5549</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/SwatchColorPicker-mount_base.html">11747</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/SwatchColorPicker-mount.html">11669</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/TagPicker-mount_base.html">2755</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/TagPicker-mount.html">2698</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/TeachingBubble-mount_base.html">13581</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/TeachingBubble-mount.html">13588</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Text-mount_base.html">466</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Text-mount.html">491</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/TextField-mount_base.html">1454</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/TextField-mount.html">1500</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/ThemeProvider-mount_base.html">1293</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/ThemeProvider-mount.html">1275</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/ThemeProvider-virtual-rerender_base.html">671</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/ThemeProvider-virtual-rerender.html">651</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1953</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1984</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Toggle-mount_base.html">843</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/Toggle-mount.html">874</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/buttonNative-mount_base.html">158</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21702/perf-test/buttonNative-mount.html">161</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

## Perf Analysis (`@fluentui/react-northstar`)
:warning: No perf measurements available
<div id="perfComment9423"/>

File: packages/react-avatar/SPEC.md (hunk 1 above)
Comment (behowell) [reply packages/react-avatar/SPEC.md:9]: Good catch, thanks. I meant to have the link point to master, and pasted the wrong one.

File: packages/react-avatar/MIGRATION.md [hunk 4]
```diff
@@ -1,51 +1,54 @@
 # Avatar Migration
 
-## STATUS: WIP 🚧
-
-This Migration guide is a work in progress and is not yet ready for use.
-
 ## Migration from v8
 
-The existing `Persona` control supports many more props than the `Avatar` control: notably the extra display text to the side of the image. As such, Avatar is only a direct replacement for Persona in cases where it was only being used to display the image (`hidePersonaDetails` is true). There is a plan to create a wrapper component EntityLayout (final name TBD) that supports placing text next to an Avatar, which is still being designed.
+The existing `Persona` control supports many more props than the `Avatar` control: notably the extra display text to the side of the image. As such, Avatar is only a direct replacement for Persona in cases where it was only being used to display the image (`hidePersonaDetails` is true). The Avatar will need to be used in conjunction with another component that displays additional personal details in order to be a full replacement for `Persona`.
```
Comment (spmonahan) [inline packages/react-avatar/MIGRATION.md:5]: Super nit pick:

```suggestion
The existing `Persona` control supports many more props than the `Avatar` control: notably the extra display text to the side of the image. As such, Avatar is only a direct replacement for Persona in cases where it was only being used to display the image (`hidePersonaDetails` is true). To fully replace `Persona` Avatar needs to be used in conjunction with another component that displays additional personal details.
```

File: packages/react-avatar/MIGRATION.md [hunk 5]
```diff
@@ -1,51 +1,54 @@
...
+- `imageInitials` => `initials`
+- `presence` => `badge` (see the `PresenceBadge` component for more details)
+  - `PersonaPresence.none` => (Default)
+  - `PersonaPresence.away` => `badge={{ status: 'away' }}`
+  - `PersonaPresence.blocked` => Not Supported.
+  - `PersonaPresence.busy` => `badge={{ status: 'busy' }}`
+  - `PersonaPresence.dnd` => `badge={{ status: 'doNotDisturb' }}`
+  - `PersonaPresence.offline` => `badge={{ status: 'offline' }}`
+  - `PersonaPresence.online` => `badge={{ status: 'available' }}`
+- `presenceColors` => Not Supported. The badge could be restyled using CSS. E.g. `badge={{ style: { color: '...' } }}`
```
Comment (spmonahan) [inline packages/react-avatar/MIGRATION.md:34]: Nit: Change "The badge could be restyled..." to "The badge can be restyled ...".

File: packages/react-avatar/MIGRATION.md [hunk 6]
```diff
@@ -1,51 +1,54 @@
...
+- `presenceTitle` => Not Supported.
+- `isOutOfOffice` => `badge={{ status: ..., outOfOffice: true }}`
+- `showUnknownPersonaCoin` => Not Supported. An approximation is: `icon={<QuestionRegular />}`
+- `initialsColor` => `color="colorful"`, or specify a color by name like `color="darkRed"`
+- `showInitialsUntilImageLoad` => This is the default behavior of Avatar.
```
Comment (spmonahan) [inline packages/react-avatar/MIGRATION.md:39]: As someone migrating it's not clear to me what, if any, action I might need to take for this prop. Is it not supported in v9?

File: packages/react-avatar/MIGRATION.md [hunk 7]
```diff
@@ -1,51 +1,54 @@
...
+- `imageShouldFadeIn` => Not Supported. Add a CSS class to the image if desired: `image={{ className: 'myFadeInClass' }}`
+- `imageShouldStartVisible` => This is the default behavior of Avatar.
```
Comment (spmonahan) [inline packages/react-avatar/MIGRATION.md:41]: I have the same comment as I did for `showInitialsUntilImageLoad` here.

File: packages/react-avatar/SPEC.md [hunk 8]
```diff
@@ -35,25 +35,34 @@ Both components support showing a badge to indicate status, but they have differ
...
+Display a user's initials:
 
 ```jsx
 <Avatar name="Miguel Garcia" />
-<Avatar size={72} name="Mona Kane" image="./MonaKane.jpg" />
-<Avatar shape="square" icon={<IDBadgeIcon />} />
 ```
 
-Displaying a badge (\*\*subject to change pending final spec for the `Badge` component):
+Display an image of a user:
```
Comment (spmonahan) [inline packages/react-avatar/SPEC.md:44]: Nit: we should be consistent and reword this to "Display a user's image" or "Display user image"

File: packages/react-avatar/SPEC.md [hunk 9]
```diff
@@ -104,97 +88,112 @@ From [Avatar.types.tsx](https://github.com/microsoft/fluentui/blob/master/packag
...
+   * @defaultvalue unset
+   */
+  active?: 'active' | 'inactive' | 'unset';
+
+  /**
+   * The appearance when `active="active"`
+   *
+   * @defaultvalue ring
+   */
+  activeAppearance?: 'ring' | 'shadow' | 'ring-shadow';
```
Comment (spmonahan) [inline packages/react-avatar/SPEC.md:178]: I noticed that we tend to use camel case in string unions rather than hyphens. Should "ring-shadow" be "ringShadow" here?

**Review by spmonahan — APPROVED** [review]

File: packages/react-avatar/SPEC.md (hunk 9 above)
Comment (behowell) [reply packages/react-avatar/SPEC.md:178]: Yeah that's a good point; we have some inconsistency in the library about camelCase vs. snake-case for attribute values. (Tooltip and Menu's `positioning` attributes use snake-case like `positioning="above-start"` as well). Given that we're already in RC, it might be too late to fix this up.

Comment (spmonahan) [reply packages/react-avatar/SPEC.md:178]: > Given that we're already in RC, it might be too late to fix this up.

Agreed. This is probably a good topic to discuss in a tech sync or RFC so we can have greater consistency going forward.
